app/main.py

```diff
@@ -1 +1,52 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self, name: str, appetite: int, is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            if self.appetite != 0:
+                print(f"Eating {self.appetite} food points...")
+                self.is_hungry = not self.is_hungry
+                return self.appetite
+            return 0
+        return 0
+
+
+class Cat(Animal):
+
+    def __init__(
+            self, name: str, appetite: int = 3, is_hungry: bool = True
+    ) -> None:
+        super().__init__(name, appetite, is_hungry)
+        self.appetite = appetite
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(
+            self, name: str, appetite: int = 7, is_hungry: bool = True
+    ) -> None:
+        super().__init__(name, appetite, is_hungry)
+        self.appetite = appetite
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list) -> int:
+    points = []
```

Comment:
you don't really need to create a list to calculate a sum. try to do it using just a number

Reply:
I don't understand how to do that in another way, it doesn' t work like this:
 # points = 0
    # for animal in animals:
    #     points += animal.feed()
    #     return points
I refactored the code as a one-liner: return sum(animal.feed() for animal in animals)